Source/Smocks/Exceptions/VerificationException.cs

```diff
@@ -60,7 +61,22 @@ private static string GetMessage(IEnumerable<IInternalSetupBase> notMatchedSetup
         {
             ArgumentChecker.NotNull(notMatchedSetups, () => notMatchedSetups);
 
-            return "Verification failed";
+            StringBuilder message = new StringBuilder();
+            foreach (IInternalSetup setup in notMatchedSetups)
+            {
+                if (!string.IsNullOrEmpty(message.ToString()))
```

Comment:
Please use curly braces for all if statements, also those with single line bodies